book/src/engine/context_global.md

```diff
@@ -33,8 +33,8 @@ let engine: Engine<AppGlobalContext, ()> = Engine::new(config, db)
 #### 3. Use GlobalContext in a resolver
 
 ```rust
-fn resolve(context: ResolverContext<AppGlobalContext, ()>) -> ExecutionResult {
-    let global_ctx = context.get_global_context()?;
+fn resolve(facade: ResolverFacade<AppGlobalContext, ()>) -> ExecutionResult {
```

Comment:
This might be super nitpicky, but I don't love the name `ResolverFacade`. If I was a stranger and saw that name, I think I might be puzzled as to what it meant. Not vetoing it, but I want to think about it.

Reply:
Please be nit picky. I have no ego about code.

I went with the name `ResolverFacade` because the intent of the code seemed to be to provide a simplified developer experience over raw interaction with the surrounding Warpgrapher code, thus matching the [definition of a facade as a design pattern](https://en.wikipedia.org/wiki/Facade_pattern).  That said, we're talking about terminology that was published in 1994, so if it's too outdated to be recognizable to a developer today, I'm fine with changing it to something else.

I would like to avoid calling it a context, though.  Having a `GraphQLContext` that comprises a `GlobalContext` and a `RequestContext` is a tolerable level of terminology reuse, because they all have the same purpose -- to convey data from outside the request handling code into the recursive structure of resolver calls.  What was called `ResolverContext` really has a different purpose: simplifying interaction with the rest of the Warpgrapher framework.  The idea that you'd make a call on your context, to get a context, from which you get a context (Resolver, GraphQL, and Global or Request, respectively) is where I had trouble.